### ISSUE TITLE
allow to move the map in the print preview

### ISSUE THREAD

**Issue by landryb** [geosolutions-it/MapStore2#8500]
## Description
Right now in the print preview widget, a static map is displayed, with a scale selectbox. That map changes if the user changes the main map, or changes the scale.

So, to recenter on a feature more precisely, the user needs to move the main map, trying to move the preview map does nothing. It's an error prone process, while it would be much more straightforward to directly change the extent by panning in the preview map.

I've looked at the print plugin doc on https://mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.Print and it doesnt seem to be a feature hidden behind a flag. There's `enableRefresh` but that's to allow the preview map to be refreshed if the main map moves.

**What kind of improvement you want to add?** (check one with "x", remove the others)
- [x] Minor changes to existing features
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**Comment by landryb** [comment]
@MaelREBOUX @catmorales feedback welcome on this.

**Comment by catmorales** [comment]
@landryb  I agree with that. enhancement.  Many users find the current operation inconvenient.
